src/main/webapp/go.js

```diff
@@ -286,10 +286,7 @@ function getAutoHint() {
  * destination, before getting a new hint.
  */
 function generatePlaceID() {
-  const request = {
-    query: hunt.getCurDestName(),
-    fields: ['place_id'],
-  };
+  const request = createRequestForPlaceID(hunt.getCurDestName());
```

Comment:
do you need this function anymore now that each place already has a placeID set at construction?